src/t-app.js

```diff
@@ -319,8 +319,8 @@ export class TApp extends TintComponent {
                             <div class="block-content">
                                 <h2>Sponsor</h2>
                                 <div id="sponsor-list">
-                                    <t-img class="sponsor" src="img/logo/tiket.jpeg" size="contain" position="center"></t-img>
-                                    <t-img class="sponsor" src="img/logo/sirclo.png" size="contain" position="center"></t-img>
+                                    <a href="//tiket.com" target="_blank"><t-img class="sponsor" src="img/logo/tiket.jpeg" size="contain" position="center"></t-img></a>
```

Comment:
add attribute `rel="noopener"` in all link with target=_blank

Reply:
@mabuak bisa di-update codenya dengan komentar @mazipan di atas?

Reply:
@mabuak  berikut tentang link target=_blank https://mathiasbynens.github.io/rel-noopener/
selain security juga mempengaruhi kinerja halaman kita jika tautan tersebut berat diload 
https://developers.google.com/web/tools/lighthouse/audits/noopener?hl=id